src/index.js

```diff
@@ -56,7 +56,9 @@ exports.pushStream = async (
           body = await Promise.resolve(transformFunction(body))
         }
         try {
-          await es.index({ index, type, id, body, refresh })
+          if (body) {
```

Comment:
Check if body is not empty object.

Reply:
Do you mean check if the variable is `{}`?

Reply:
Using something like: `Object.values(body).length >= 1` or `Object.keys(body).length >= 1` ?

Reply:
It needs to be:
`Object.keys(obj).length === 0 && obj.constructor === Object`

Reply:
If you could also update a README file that if transform function returns false or empty object index won't be triggered. Thx.

Reply:
Fixed.